### PR TITLE
CC-9:  Input Form

### PR DESCRIPTION
**Pull request by Sarahunt89**
Created labels and input fields.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 import React from 'react';
+
+import InputForm from './components/InputForm/InputForm.jsx';
+
+import './App.css';
+
+function App() {
+  return (
+    <InputForm />
+
```
Comment (jdickerson-banyan) [inline src/App.js:10]: Why did you change this file? I understand you might render your component in this file in order to test the functionality, but it should never have been added to your commit. This would overwrite the current App.js.

File: src/components/InputForm/style.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+`;
+
+export const Label = styled.label`
+    font-size: 20px;
+    display: flex;
+    padding-right: 20%;
+    padding-left: 20%;
+    flex-direction: column;
+    margin-bottom: 30px;
+
```
Comment (jdickerson-banyan) [inline src/components/InputForm/style.js:22]: Please remove this blank line. Try running the command 'npm run format', it should remove it for you (and any other formatting mistakes)

File: src/components/InputForm/style.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    padding: 12px;
+    border: 1px solid #ccc;
+    resize: none;
+    border-width: 2px;
+    border-style: inset;
+    border-color: (rgb(118, 118, 118), rgb(133, 133, 133));
+`;
+
+
+
```
Comment (jdickerson-banyan) [inline src/components/InputForm/style.js:47]: Extra lines need to be removed

File: src/components/InputForm/InputForm.jsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+
+import { Container, Label, Input, Textarea, Button, } from './style';
```
Comment (jdickerson-banyan) [inline src/components/InputForm/InputForm.jsx:4]: Good job with the styled components!

File: src/components/InputForm/InputForm.jsx [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    const [description, setDescription] = useState('');
+    const [findings, setFindings] = useState('');
+    const [discussion, setDiscussion] = useState('');
+    const [conclusion, setConclusion] = useState('');
+    const [recommendations, setRecommendations] = useState('');
+    const [implementation, setImplementation] = useState('');
+    const [references, setReferences] = useState('');
+    const [appendices, setAppendices] = useState('');
+
+
```
Comment (jdickerson-banyan) [inline src/components/InputForm/InputForm.jsx:18]: Extra line, please remove

File: src/components/InputForm/InputForm.jsx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                <form onSubmit={handleSubmit}>
+
+                    <Label>
+                        <p className="input-text">Project Name:</p>
+                        <Input
+                            type="text"
+                            id="projectName"
+                            name="projectName"
+                            onChange={event => setProjectName(event.target.value)} required />
+                    </Label>
```
Comment (jdickerson-banyan) [inline src/components/InputForm/InputForm.jsx:37]: While I am sure this logic works, each of these Label sections are actually repeated code that could be pulled into a reusable component. Let's do that instead.

**Review by jdickerson-banyan — CHANGES_REQUESTED** [review]
Great job so far!

File: src/App.js (hunk 1 above)
Comment (Sarahunt89) [reply src/App.js:10]: Fixed this by copy and paste the correct code.

File: src/components/InputForm/style.js (hunk 2 above)
Comment (Sarahunt89) [reply src/components/InputForm/style.js:22]: I did an 'npm run format', so that should take care of that,

File: src/components/InputForm/style.js (hunk 3 above)
Comment (Sarahunt89) [reply src/components/InputForm/style.js:47]: I did an 'npm run format', so that should take care of that.

File: src/components/InputForm/InputForm.jsx (hunk 4 above)
Comment (Sarahunt89) [reply src/components/InputForm/InputForm.jsx:4]: Thank you

File: src/components/InputForm/InputForm.jsx (hunk 5 above)
Comment (Sarahunt89) [reply src/components/InputForm/InputForm.jsx:18]: I did an 'npm run format', so that should take care of that.

File: src/components/InputForm/InputForm.jsx (hunk 6 above)
Comment (Sarahunt89) [reply src/components/InputForm/InputForm.jsx:37]: Made the input form reusable.

**Comment by Sarahunt89** [comment]
CC-9: Input Form
Created a reusable input form.
InputFormScreenshot

**Review by jdickerson-banyan — APPROVED** [review]
